plugins/slack_app/app/controllers/slack_app_controller.rb

```diff
@@ -53,7 +53,7 @@ def command_response
     return unknown_user unless @current_user_from_slack
 
     # Parse the command
-    project_name, ref_name, stage_name = @payload['text'].match(/([^\s\/]+)(?:\/(\S+))?\s*(?:to\s+(.*))?/).captures
+    project_name, ref_name, stage_name = @payload['text'].scan(/([^\s\/]+)(?:\/(\S+))?\s*(?:to\s+(.*))?/).flatten.compact
```

Comment:
should avoid combining multiple matches, so add a `.first`
```suggestion
    project_name, ref_name, stage_name = @payload['text'].scan(/([^\s\/]+)(?:\/(\S+))?\s*(?:to\s+(.*))?/).first.flatten.compact
```